src/lib.rs

```diff
@@ -376,6 +377,29 @@ impl ListValue {
     }
 }
 
+impl Hash for qdrant::PointId {
+    fn hash<H: Hasher>(&self, state: &mut H) {
+        use qdrant::point_id::PointIdOptions::{Num, Uuid};
+        match &self.point_id_options {
+            Some(Num(u)) => state.write_u64(*u),
+            Some(Uuid(s)) => s.hash(state),
+            _ => {}
```

Comment:
Nitpick, but maybe we can make this exhaustive:

```suggestion
            None => {}
```